### ISSUE TITLE
Intro Screens & Minimum Requirements of Settings

### ISSUE THREAD

**Issue by angelsix** [angelsix/blueberry#5]
So the purpose here is to define the entire requirements for the intro/welcome screens of the app. 
This should include what pages we need, whats on those pages, and from that, what settings are we extracting.
I think already we have:
 - mmol/L or mg/dL (what is technical name for this option)
 - 12/24 hour clock
 - Name (to give personal messages in app for motivation)
Is there anything else absolutely needed for the intro screens?

**Comment by Krellon** [comment]
I think a history screen. Shows selectable history ranges for Day/Week/Month and year. Good to see your glucose trend. This could be a line or bar graph. This could be the app landing page. PS: I am not diabetic so basing my opinion on what i would like if i was diabetic.

**Comment by angelsix** [comment]
Would we want the history being its own screen? I think so, but if so I think that should be after initial reelase, and for now, the overview (home) page should have a basic left right swipe to move to previous and next dates, and maybe a button to open a calendar to pick a specific date, then the history page in the future could be much more details?

**Comment by Krellon** [comment]
Own screen for sure. Regarding presentation. I like the overview idea, eg, line graph of reading over date with a vertical line through it. The line is a scrollable cursor controlled by sliding finger left/right. You have a details button which allows the user to then present more detailed measurement data on a separate screen for the selected result. The overview allows a user/doctor to see trend in an instant.

**Comment by Krellon** [comment]
You could have a reminder feature where the user sets a reminder for when to take a  measurement (assuming there taken at regular intervals) The home screen could then have a little bell indicting the reminder alarm is set/unset. A user would tap the bell for timer settings.

**Comment by angelsix** [comment]
Yes I think both of them are good and future additions from milestone 1.0. I'll make future discussions on those two topics

**Comment by angelsix** [comment]
> Own screen for sure. Regarding presentation. I like the overview idea, eg, line graph of reading over date with a vertical line through it. The line is a scrollable cursor controlled by sliding finger left/right. You have a details button which allows the user to then present more detailed measurement data on a separate screen for the selected result. The overview allows a user/doctor to see trend in an instant.

https://github.com/angelsix/blueberry/issues/12

**Comment by Robert-git-hub** [comment]
mmol/L or mg/dL - blood glucose concentration
